pkg/kubernetes/scripts/projects.js

```diff
@@ -940,10 +977,13 @@
 
                 return methods.check(request, { "metadata.name": "#project-new-name" })
                     .then(function() {
-                        return methods.create(request)
-                            .then(function() {
-                                return projectData.shareImages(name, fields.access === "shared");
-                            });
+                        return methods.create(request);
+                    })
+                    .then(function(){
+                        return projectData.shareImages(name, fields.access === "shared");
+                    })
+                    .then(function(){
+                        return projectData.makeRegistryAdmin(name, dialogData.currUser);
```

Comment:
> for time being we need this to make registry-admin useful in our UI
> 
> @aweiteka IMO we need to add a project template which will not create a 'admin' policy. ok ?

At this point I'm much less concerned about if the user is admin or registry-admin. The UI should reflect that they are "Admin" either way. Also, we want them to be assigned role "registry-admin" role when new admin users are added to a project. That's really it. Then it will work regardless of  the project template we use.